src/app/global/local_account_settings.nim

```diff
@@ -33,8 +34,15 @@ QtObject:
   proc setFileName*(self: LocalAccountSettings, fileName: string) =
     if(not self.settings.isNil):
       self.settings.delete
-    let filePath = os.joinPath(self.settingsFileDir, fileName)
-    self.settings = newQSettings(filePath, QSettingsFormat.IniFormat)
+    let
+      currentFilePath = os.joinPath(self.settingsFileDir, self.currentFileName)
+      newFilePath = os.joinPath(self.settingsFileDir, fileName)
+    self.currentFileName = fileName
+    try:
+      moveFile(currentFilePath, newFilePath)
```

Comment:
may `if currentFilePath != newFilePath`: be useful here?

Reply:
@MishkaRogachev the way how we use it, not needed, cause even if the source and the destination path are the same that doesn't result in an error, but anyhow I added the guard you suggested.